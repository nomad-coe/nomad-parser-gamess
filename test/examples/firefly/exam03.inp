! EXAM03.
!
! 3-B-1 CH2  ROHF calculation on methylene ground state.
! The wavefunction is a pure triplet state (<S**2> = 2),
! and so has a higher energy than the second example.
!
! This job tests the ROHF wavefunction and gradient code.
! It also tests the direct SCF procedure.
!
! The initial energy is -37.228465067.
! The FINAL energy is -37.2778767104 after 7 iterations.
! Mulliken, Lowdin charges on C are -0.020346, 0.019470.
! The Hydrogen atom spin density is 0.0129735
! The dipole moment is 0.025099 Debye. 
! The RMS gradient is 0.027505564
!
 $CONTRL SCFTYP=ROHF MULT=3 RUNTYP=GRADIENT $END
 $SYSTEM TIMLIM=1 MEMORY=100000 $END
 $BASIS  GBASIS=STO NGAUSS=2 $END
 $GUESS  GUESS=HUCKEL $END
 $SCF    DIRSCF=.TRUE. $END
 $DATA
Methylene...3-B-1 state...ROHF/STO-2G
Cnv  2

Carbon     6.0
Hydrogen   1.0   -0.82884  0.0   0.7079  
 $END
