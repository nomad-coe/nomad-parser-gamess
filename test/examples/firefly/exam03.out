                             Running on host I7

          ******************************************************
          *      Firefly version 8.2.0, build number 10203     *
          *   Compiled on    Friday,    16-09-2016, 21:20:07   *
          *Code development and Intel/AMD specific optimization*
          *  Copyright (c) 1994, 2016 by  Alex A. Granovsky,   *
          *          Firefly Project, Moscow, Russia.          *
          *   Some parts of this program include code due to   *
          * work of Jim Kress, Peter Burger, and Robert Ponec. *
          ******************************************************
          *             Firefly Project homepage:              *
          * http://classic.chem.msu.su/gran/firefly/index.html *
          *                      e-mail:                       *
          *               gran@classic.chem.msu.su             *
          *This program is not a free software and is provided *
          *exclusively to its registered users under the terms *
          *       of Firefly package license agreement         *
          * Unauthorized use of Firefly is strongly prohibited *
          *   This program may not be redistributed without    *
          * the specific, written permission of its developers.*
          ******************************************************

          ******************************************************
          *  PARTIALLY BASED ON  US GAMESS VERSION 6 JUN 1999, *
          *   US GAMESS VERSIONS  6 SEP 2001 AND 12 DEC 2003   *
          *             FROM IOWA STATE UNIVERSITY             *
          * M.W.SCHMIDT, K.K.BALDRIDGE, J.A.BOATZ, S.T.ELBERT, *
          *   M.S.GORDON, J.H.JENSEN, S.KOSEKI, N.MATSUNAGA,   *
          *          K.A.NGUYEN, S.J.SU, T.L.WINDUS,           *
          *       TOGETHER WITH M.DUPUIS, J.A.MONTGOMERY       *
          *         J.COMPUT.CHEM.  14, 1347-1363(1993)        *
          ******************************************************


 Core i7    / Win32  Firefly version running under Windows NT
 Running on Intel CPU:  Brand ID  0, Family  6, Model  26, Stepping  4
 CPU Brand String    :  Intel(R) Core(TM) i7 CPU         940  @ 2.93GHz 
 CPU Features        :  x87 FPU, CMOV, MMX, SSE, SSE2, SSE3, SSSE3, SSE4.1, SSE4.2, HTT, EM64T
 Data cache size     :  L1 32 KB, L2  256 KB, L3  8192 KB
 max    # of   cores/package :   8
 max    # of threads/package :  16
 max     cache sharing level :  16
 actual # of   cores/package :   4
 actual # of threads/package :   8
 actual # of threads/core    :   2
 Operating System successfully passed SSE support test.


 PARALLEL VERSION (UNIFIED) RUNNING IN SERIAL MODE USING SINGLE PROCESS

 EXECUTION OF FIREFLY BEGUN 16:58:53 17-SEP-2016    

            ECHO OF THE FIRST FEW INPUT CARDS -
 INPUT CARD>! EXAM03.                                                                       
 INPUT CARD>!                                                                               
 INPUT CARD>! 3-B-1 CH2  ROHF calculation on methylene ground state.                        
 INPUT CARD>! The wavefunction is a pure triplet state (<S**2> = 2),                        
 INPUT CARD>! and so has a higher energy than the second example.                           
 INPUT CARD>!                                                                               
 INPUT CARD>! This job tests the ROHF wavefunction and gradient code.                       
 INPUT CARD>! It also tests the direct SCF procedure.                                       
 INPUT CARD>!                                                                               
 INPUT CARD>! The initial energy is -37.228465067.                                          
 INPUT CARD>! The FINAL energy is -37.2778767104 after 7 iterations.                        
 INPUT CARD>! Mulliken, Lowdin charges on C are -0.020346, 0.019470.                        
 INPUT CARD>! The Hydrogen atom spin density is 0.0129735                                   
 INPUT CARD>! The dipole moment is 0.025099 Debye.                                          
 INPUT CARD>! The RMS gradient is 0.027505564                                               
 INPUT CARD>!                                                                               
 INPUT CARD> $CONTRL SCFTYP=ROHF MULT=3 RUNTYP=GRADIENT $END                                
 INPUT CARD> $SYSTEM TIMLIM=1 MEMORY=100000 $END                                            
 INPUT CARD> $BASIS  GBASIS=STO NGAUSS=2 $END                                               
 INPUT CARD> $GUESS  GUESS=HUCKEL $END                                                      
 INPUT CARD> $SCF    DIRSCF=.TRUE. $END                                                     
 INPUT CARD> $DATA                                                                          
 INPUT CARD>Methylene...3-B-1 state...ROHF/STO-2G                                           
 INPUT CARD>Cnv  2                                                                          
 INPUT CARD>                                                                                
 INPUT CARD>Carbon     6.0                                                                  
 INPUT CARD>Hydrogen   1.0   -0.82884  0.0   0.7079                                         
 INPUT CARD> $END                                                                           
     400000 WORDS OF MEMORY AVAILABLE

 This job is executing on     1 unique host(s)
 Minimum number of processes per host is:    1
 Maximum number of processes per host is:    1

 On master's host, detected    8 CPU core(s) in aggregate

 Warning: HTT is enabled, bitmask of physically unique cores is 0x000000AA

 SMT aware parts of program will use              2 threads.

 Creating thread pool to serve up to            128 threads.

 Activating Call64 option.

 Using 64-bit DGEMM by default.

     BASIS OPTIONS
     -------------
     GBASIS=STO          IGAUSS=       2      POLAR=NONE    
     NDFUNC=       0     NFFUNC=       0     DIFFSP=       F
     NPFUNC=       0      DIFFS=       F


     RUN TITLE
     ---------
 Methylene...3-B-1 state...ROHF/STO-2G                                           

 THE POINT GROUP OF THE MOLECULE IS CNV     
 THE ORDER OF THE PRINCIPAL AXIS IS     2

 ATOM      ATOMIC                      COORDINATES (BOHR)
           CHARGE         X                   Y                   Z
 CARBON      6.0     0.0000000000        0.0000000000        0.0000000000
 HYDROGEN    1.0     1.5662806018        0.0000000000        1.3377371242
 HYDROGEN    1.0    -1.5662806018        0.0000000000        1.3377371242

          INTERNUCLEAR DISTANCES (ANGS.)
          ------------------------------

                    CARBON         HYDROGEN       HYDROGEN  

  1  CARBON          0.0000000      1.0899992 *    1.0899992 *  
  2  HYDROGEN        1.0899992 *    0.0000000      1.6576800 *  
  3  HYDROGEN        1.0899992 *    1.6576800 *    0.0000000    

  * ... LESS THAN  3.000


     ATOMIC BASIS SET
     ----------------
 THE CONTRACTED PRIMITIVE FUNCTIONS HAVE BEEN UNNORMALIZED
 THE CONTRACTED BASIS FUNCTIONS ARE NOW NORMALIZED TO UNITY

 SHELL TYPE PRIM    EXPONENT          CONTRACTION COEFFICIENTS

 CARBON    

   1   S    1      27.385033    3.669807 (  0.430128) 
   1   S    2       4.874522    1.587353 (  0.678914) 

   2   L    3       1.136748    0.038816 (  0.049472)     0.855856 (  0.511541) 
   2   L    4       0.288309    0.270261 (  0.963782)     0.184542 (  0.612820) 

 HYDROGEN  

   4   S    5       1.309756    0.375320 (  0.430128) 
   4   S    6       0.233136    0.162342 (  0.678914) 

 TOTAL NUMBER OF SHELLS              =    4
 TOTAL NUMBER OF BASIS FUNCTIONS     =    7
 NUMBER OF ELECTRONS                 =    8
 CHARGE OF MOLECULE                  =    0
 STATE MULTIPLICITY                  =    3
 NUMBER OF OCCUPIED ORBITALS (ALPHA) =    5
 NUMBER OF OCCUPIED ORBITALS (BETA ) =    3
 TOTAL NUMBER OF ATOMS               =    3
 THE NUCLEAR REPULSION ENERGY IS        6.1450362778

     $CONTRL OPTIONS
     ---------------
     SCFTYP=ROHF         RUNTYP=GRADIENT     EXETYP=RUN     
     MPLEVL=       0     LOCAL =NONE         UNITS =ANGS    
     MULT  =       3     ICHARG=       0     MAXIT =      30
     NPRINT=       7     IREST =       0     COORD =UNIQUE  
     ECP   =NONE         NORMF =       0     NORMP =       0
     ITOL  =      20     ICUT  =       9     NZVAR =       0
     NOSYM =       0     INTTYP=POPLE        GEOM  =INPUT   
     PLTORB=       F     MOLPLT=       F     RPAC  =       F
     AIMPAC=       0     FRIEND=             CITYP =NONE    
     DFTTYP=NONE    

     $SYSTEM OPTIONS
     ---------------
     KDIAG =       0     MEMORY=   400000     TIMLIM=       60.0 SEC.
     COREFL=       F     PTIME =        F     XDR   =       F
     BALTYP=LOOP         CBASE = 00401000     FBASE = 63E110C0

          ----------------
          PROPERTIES INPUT
          ----------------

     MOMENTS            FIELD           POTENTIAL          DENSITY
 IEMOM =       1   IEFLD =       0   IEPOT =       0   IEDEN =       0
 WHERE =COMASS     WHERE =NUCLEI     WHERE =NUCLEI     WHERE =NUCLEI  
 OUTPUT=BOTH       OUTPUT=BOTH       OUTPUT=BOTH       OUTPUT=BOTH    
 IEMINT=       0   IEFINT=       0                     IEDINT=       0
                                                       MORB  =       0

          EXTRAPOLATION IN EFFECT

          ----------------------
          INTEGRAL INPUT OPTIONS
          ----------------------
 NOPK  =       1 NORDER=       0 SCHWRZ=       T

 ATTENTION! AO INTEGRALS WILL BE PACKED.
 THRESHOLD FOR PACKING PKTHR =  0.10000000D-01

     -------------------------------
     INTEGRAL TRANSFORMATION OPTIONS
     -------------------------------
     NWORD  =       0     CUTTRF = 1.0E-09
     MPTRAN =       0     DIRTRF =       T
     AOINTS =DUP          IREST  =       0

     ------------------------------------------
     THE POINT GROUP IS CNV, NAXIS= 2, ORDER= 4
     ------------------------------------------

     DIMENSIONS OF THE SYMMETRY SUBSPACES ARE
 A1  =   4      A2  =   0      B1  =   2      B2  =   1

 ..... DONE SETTING UP THE RUN .....

 CPU        TIME:   STEP =      0.02 ,  TOTAL =        0.0 SECONDS (    0.0 MIN)
 WALL CLOCK TIME:   STEP =      0.02 ,  TOTAL =        0.0 SECONDS (    0.0 MIN)
 CPU UTILIZATION:   STEP =     97.60%,  TOTAL =     195.18%

                    --------------------------------
                    SINGLE POINT ENERGY AND GRADIENT
                    --------------------------------

          ********************
          1 ELECTRON INTEGRALS
          ********************
 ...... END OF ONE-ELECTRON INTEGRALS ......

 CPU        TIME:   STEP =      0.00 ,  TOTAL =        0.0 SECONDS (    0.0 MIN)
 WALL CLOCK TIME:   STEP =      0.00 ,  TOTAL =        0.0 SECONDS (    0.0 MIN)
 CPU UTILIZATION:   STEP =      0.00%,  TOTAL =     191.36%

          -------------
          GUESS OPTIONS
          -------------
          GUESS =HUCKEL            NORB  =       0          NORDER=       0
          MIX   =       F          PRTMO =       F          SYMDEN=       F
          TOLZ  = 0.0E+00          TOLE  = 0.0E+00

 INITIAL GUESS ORBITALS GENERATED BY HUCKEL   ROUTINE.
 HUCKEL GUESS REQUIRES      2471 WORDS.

 LOG10 OF CONDITION NUMBER OF OVERLAP MATRIX IS:  9.82212306E-01

 SYMMETRIES FOR INITIAL GUESS ORBITALS FOLLOW.  ALPHA SET(S).
     5 ORBITALS ARE OCCUPIED (    1 CORE ORBITALS).
     2=A1       3=B1       4=A1       5=B2       6=B1       7=A1  

 SYMMETRIES FOR INITIAL GUESS ORBITALS FOLLOW.   BETA SET(S).
     3 ORBITALS ARE OCCUPIED (    1 CORE ORBITALS).
     2=A1       3=B1       4=A1       5=B2       6=B1       7=A1  
 ...... END OF INITIAL ORBITAL SELECTION ......

 CPU        TIME:   STEP =      0.00 ,  TOTAL =        0.0 SECONDS (    0.0 MIN)
 WALL CLOCK TIME:   STEP =      0.00 ,  TOTAL =        0.0 SECONDS (    0.0 MIN)
 CPU UTILIZATION:   STEP =      0.00%,  TOTAL =     164.57%

          --------------------
          2 ELECTRON INTEGRALS
          --------------------

 DIRECT SCF METHOD SKIPS INTEGRAL STORAGE ON DISK.
 DIRECT TRANSFORMATION SKIPS AO INTEGRAL STORAGE ON DISK.
 SCHWARZ INEQUALITY OVERHEAD:        28 INTEGRALS, CPU TIME=        0.00
 ...... END OF TWO-ELECTRON INTEGRALS .....

 CPU        TIME:   STEP =      0.00 ,  TOTAL =        0.0 SECONDS (    0.0 MIN)
 WALL CLOCK TIME:   STEP =      0.00 ,  TOTAL =        0.0 SECONDS (    0.0 MIN)
 CPU UTILIZATION:   STEP =      0.00%,  TOTAL =     152.00%

          --------------------
          ROHF SCF CALCULATION
          --------------------

     NUCLEAR ENERGY =         6.1450362778
     MAXIT = 30     NPUNCH=  2     MULT=  3
     EXTRAP=T  DAMP=F  SHIFT=F  RSTRCT=F  DIIS=F  SOSCF=F
     DENSITY CONV=  2.00E-05
     ROHF CANONICALIZATION PARAMETERS
             C-C     O-O     V-V
     ALPHA -0.5000  0.5000  1.5000
     BETA   1.5000  0.5000 -0.5000
     MEMORY REQUIRED FOR UHF/ROHF STEP=     15478 WORDS.

 DIRECT SCF CALCULATION, SCHWRZ=T   FDIFF=T
 SCHWARZ INEQUALITY OVERHEAD:        28 INTEGRALS, CPU TIME=        0.00

                                                                           NONZERO    BLOCKS
 ITER EX   TOTAL ENERGY      E CHANGE  DENSITY CHANGE    DIIS ERROR      INTEGRALS   SKIPPED
   1  0   -37.228465067   -37.228465067   0.358163851   0.000000000            322        52
   2  1   -37.277668927    -0.049203860   0.022470485   0.000000000            322        52
   3  2   -37.277870051    -0.000201124   0.003124254   0.000000000            322        52
   4  3   -37.277876442    -0.000006391   0.000738118   0.000000000            322        52
   5  0   -37.277876699    -0.000000258   0.000165742   0.000000000            322        52
   6  1   -37.277876710    -0.000000011   0.000005988   0.000000000            322        52
   7  2   -37.277876710     0.000000000   0.000001240   0.000000000            322        52

          -----------------
          DENSITY CONVERGED
          -----------------
     TIME TO FORM FOCK OPERATORS =       0.00 SECONDS (       0.00 SEC/ITER)
     FOCK TIME ON FIRST ITERATION=       0.00, LAST ITERATION=       0.00
     TIME TO SOLVE SCF EQUATIONS =       0.00 SECONDS (       0.00 SEC/ITER)

 FINAL ENERGY IS      -37.2778767104 AFTER   7 ITERATIONS

          --------------------
          SPIN SZ   =    1.000
          S-SQUARED =    2.000
          --------------------

          ------------
          EIGENVECTORS
          ------------

                      1          2          3          4          5
                  -10.5439    -0.6511    -0.4135    -0.0937     0.0249
                     A1         A1         B1         A1         B2  
    1  C   1  S  -0.984409  -0.219850   0.000000   0.195036   0.000000
    2  C   1  S  -0.061236   0.560678   0.000000  -0.722677   0.000000
    3  C   1  X   0.000000   0.000000   0.544050   0.000000   0.000000
    4  C   1  Y   0.000000   0.000000   0.000000   0.000000   1.000000
    5  C   1  Z  -0.010664   0.230124   0.000000   0.719120   0.000000
    6  H   2  S   0.016683   0.313589   0.463753   0.203447   0.000000
    7  H   3  S   0.016683   0.313589  -0.463753   0.203447   0.000000

                      6          7
                    0.6607     0.7380
                     B1         A1  
    1  C   1  S   0.000000  -0.215546
    2  C   1  S   0.000000   1.110906
    3  C   1  X  -1.079349   0.000000
    4  C   1  Y   0.000000   0.000000
    5  C   1  Z   0.000000   0.884691
    6  H   2  S   0.834073  -0.903176
    7  H   3  S  -0.834073  -0.903176
 ...... END OF ROHF CALCULATION ......

 CPU        TIME:   STEP =      0.00 ,  TOTAL =        0.0 SECONDS (    0.0 MIN)
 WALL CLOCK TIME:   STEP =      0.01 ,  TOTAL =        0.0 SECONDS (    0.0 MIN)
 CPU UTILIZATION:   STEP =      0.00%,  TOTAL =     120.46%


                        -------------------------------
                        properties for the ROHF density
                        -------------------------------

          -----------------
          ENERGY COMPONENTS
          -----------------

         WAVEFUNCTION NORMALIZATION =       1.0000000000

                ONE ELECTRON ENERGY =     -62.3586950114
                TWO ELECTRON ENERGY =      18.9357820232
           NUCLEAR REPULSION ENERGY =       6.1450362778
                                      ------------------
                       TOTAL ENERGY =     -37.2778767104

 ELECTRON-ELECTRON POTENTIAL ENERGY =      18.9357820232
  NUCLEUS-ELECTRON POTENTIAL ENERGY =     -99.3997924818
   NUCLEUS-NUCLEUS POTENTIAL ENERGY =       6.1450362778
                                      ------------------
             TOTAL POTENTIAL ENERGY =     -74.3189741809
               TOTAL KINETIC ENERGY =      37.0410974704
                 VIRIAL RATIO (V/T) =       2.0063923387

  ...... PI ENERGY ANALYSIS ......

 ENERGY ANALYSIS:
            FOCK ENERGY=     -24.4871307981
          BARE H ENERGY=     -62.3586950114
     ELECTRONIC ENERGY =     -43.4229129047
         KINETIC ENERGY=      37.0410974704
          N-N REPULSION=       6.1450362778
           TOTAL ENERGY=     -37.2778766270
        SIGMA PART(1+2)=     -41.0265967284
               (K,V1,2)=      35.6096101724     -93.5194219707      16.8832150700
           PI PART(1+2)=      -2.3963161764
               (K,V1,2)=       1.4314872981      -5.8803705111       2.0525670366
  SIGMA SKELETON, ERROR=     -34.8815604506       0.0000000000
             MIXED PART= 0.00000E+00 0.00000E+00 0.00000E+00 0.00000E+00
 ...... END OF PI ENERGY ANALYSIS ......

          ---------------------------------------
          MULLIKEN AND LOWDIN POPULATION ANALYSES
          ---------------------------------------

     MULLIKEN ATOMIC POPULATION IN EACH MOLECULAR ORBITAL

                      1          2          3          4          5

                  2.000000   2.000000   2.000000   1.000000   1.000000

    1             2.004816   1.113471   0.950987   0.951071   1.000000
    2            -0.002408   0.443264   0.524506   0.024464   0.000000
    3            -0.002408   0.443264   0.524506   0.024464   0.000000

     ATOMIC SPIN POPULATION (ALPHA MINUS BETA)
       ATOM         MULL.POP.                    LOW.POP.
    1 CARBON        1.951071                     1.962807
    2 HYDROGEN      0.024464                     0.018596
    3 HYDROGEN      0.024464                     0.018596

               ----- POPULATIONS IN EACH AO -----
                             MULLIKEN      LOWDIN
              1  C   1  S     1.98505     1.97704
              2  C   1  S     1.28465     1.19731
              3  C   1  X     0.95099     0.99330
              4  C   1  Y     1.00000     1.00000
              5  C   1  Z     0.79966     0.81288
              6  H   2  S     0.98983     1.00973
              7  H   3  S     0.98983     1.00973

          ----- MULLIKEN ATOMIC OVERLAP POPULATIONS -----
          (OFF-DIAGONAL ELEMENTS NEED TO BE MULTIPLIED BY 2)

             1           2           3

    1    5.3024020
    2    0.3589720   0.6687583
    3    0.3589720  -0.0379033   0.6687583

          TOTAL MULLIKEN AND LOWDIN ATOMIC POPULATIONS
       ATOM         MULL.POP.    CHARGE          LOW.POP.     CHARGE
    1 CARBON        6.020346   -0.020346         5.980530    0.019470
    2 HYDROGEN      0.989827    0.010173         1.009735   -0.009735
    3 HYDROGEN      0.989827    0.010173         1.009735   -0.009735

          -------------------------------
          BOND ORDER AND VALENCE ANALYSIS     BOND ORDER THRESHOLD=0.050
          -------------------------------

                   BOND                       BOND                       BOND
  ATOM PAIR DIST  ORDER      ATOM PAIR DIST  ORDER      ATOM PAIR DIST  ORDER
    1   2  1.090  0.995        1   3  1.090  0.995

                       TOTAL       BONDED        FREE
      ATOM            VALENCE     VALENCE     VALENCE
    1 CARBON            3.895       1.990       1.905
    2 HYDROGEN          1.000       0.999       0.001
    3 HYDROGEN          1.000       0.999       0.001

          ATOMIC SPIN DENSITY AT THE NUCLEUS (A.U.)
          -----------------------------------------

    1  CARBON       6.0     0.6841366
    2  HYDROGEN     1.0     0.0129735
    3  HYDROGEN     1.0     0.0129735

          ---------------------
          ELECTROSTATIC MOMENTS
          ---------------------

 POINT   1           X           Y           Z (BOHR)    CHARGE
                 0.000000    0.000000    0.192386        0.00 (A.U.)
         DX          DY          DZ         /D/  (DEBYE)
     0.000000    0.000000    0.025099    0.025099
 ...... END OF PROPERTY EVALUATION ......

 CPU        TIME:   STEP =      0.02 ,  TOTAL =        0.0 SECONDS (    0.0 MIN)
 WALL CLOCK TIME:   STEP =      0.00 ,  TOTAL =        0.0 SECONDS (    0.0 MIN)
 CPU UTILIZATION:   STEP =   1782.93%,  TOTAL =     174.79%
 ......END OF NBO ANALYSIS......

 CPU        TIME:   STEP =      0.00 ,  TOTAL =        0.0 SECONDS (    0.0 MIN)
 WALL CLOCK TIME:   STEP =      0.00 ,  TOTAL =        0.0 SECONDS (    0.0 MIN)
 CPU UTILIZATION:   STEP =      0.00%,  TOTAL =     170.75%

 BEGINNING ONE ELECTRON GRADIENT...
 ..... END OF 1-ELECTRON GRADIENT ......

 CPU        TIME:   STEP =      0.00 ,  TOTAL =        0.0 SECONDS (    0.0 MIN)
 WALL CLOCK TIME:   STEP =      0.00 ,  TOTAL =        0.0 SECONDS (    0.0 MIN)
 CPU UTILIZATION:   STEP =      0.00%,  TOTAL =     166.70%

          ----------------------
          GRADIENT OF THE ENERGY
          ----------------------
 SCHWARZ SCREENING SKIPPED          0 BLOCKS, COMPUTED         27 BLOCKS

 ...... END OF 2-ELECTRON GRADIENT ......

 CPU        TIME:   STEP =      0.00 ,  TOTAL =        0.0 SECONDS (    0.0 MIN)
 WALL CLOCK TIME:   STEP =      0.00 ,  TOTAL =        0.0 SECONDS (    0.0 MIN)
 CPU UTILIZATION:   STEP =      0.00%,  TOTAL =     151.24%

                         ----------------------
                         GRADIENT OF THE ENERGY
                         ----------------------

      ATOM                 E'X               E'Y               E'Z 
    1 CARBON           0.000000000       0.000000000      -0.057220624
    2 HYDROGEN        -0.030803449       0.000000000       0.028610312
    3 HYDROGEN         0.030803449       0.000000000       0.028610312

                   MAXIMUM GRADIENT =    0.057220624
                       RMS GRADIENT =    0.027505564
  $VIB   
          IVIB=   0 IATOM=   0 ICOORD=   0 E=      -37.2778767104
  0.000000000E+00 0.000000000E+00-5.722062398E-02-3.080344877E-02 0.000000000E+00
  2.861031199E-02 3.080344877E-02 0.000000000E+00 2.861031199E-02
  1.128770855E-15 3.668220203E-17 2.509857792E-02

 ..... END OF SINGLE POINT GRADIENT .....

 CPU        TIME:   STEP =      0.00 ,  TOTAL =        0.0 SECONDS (    0.0 MIN)
 WALL CLOCK TIME:   STEP =      0.00 ,  TOTAL =        0.0 SECONDS (    0.0 MIN)
 CPU UTILIZATION:   STEP =      0.00%,  TOTAL =     150.73%

                         I/O STATISTICS:
 DATA READ TOTAL =        0.166 MB,  DATA WRITTEN TOTAL =        0.355 MB

      396849 WORDS OF    DYNAMIC MEMORY USED
      130048 BYTES OF    HEAP MEMORY    USED,       81335 BYTES REMAIN IN USE
 EXECUTION OF FIREFLY TERMINATED NORMALLY 16:58:54 17-SEP-2016    
